pkgs/development/tools/xcodes/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, stdenv, fetchurl, unzip }:
+
+stdenv.mkDerivation rec {
+  pname = "xcodes";
+  version = "1.3.0";
+
+  src = fetchurl {
+    url = "https://github.com/XcodesOrg/xcodes/releases/download/${version}/xcodes.zip";
+    sha256 = "0cqb0gfb80xrnm4fipr46kbzqz2kicc13afhdxkbifzm4k83any5";
+  };
+
+  buildInputs = [ unzip ];
+  phases = ["unpackPhase" "installPhase"];
```

Comment:
Using `phases` is going to be deprecated (https://github.com/NixOS/nixpkgs/issues/28910), you should use `dontUnpack` and `dontInstall` instead.